### ISSUE TITLE
Distinguish between data and background layers

### ISSUE THREAD

**Issue by jidanni** [MarcusWolschon/osmeditor4android#1752]
![Screenshot_20220905-200806_Vespucci.jpg](https://user-images.githubusercontent.com/1230959/188522595-55f62946-26cc-46a7-8b5b-a8b804cf1dcb.jpg)
Here the user has selected openstreetmap  as a background. This creates two layers with the same name. This is very confusing. So please distinguish them one way or the other; thank you. Another problem is this thing is truncated even in horizontal view. Also this thing needs a title at the top: the title should be Layers. Thank you.

**Comment by jidanni** [comment]
It seems one of them should be called background, and the other should be called data. Okay perhaps use B: and D:, that would give enough room for showing the currently selected item that each one is using. (However the data one pretty much always uses the same data anyway..)

**Comment by simonpoole** [comment]
Duplicate of https://github.com/MarcusWolschon/osmeditor4android/issues/1145
